outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
Ooh, so according to ben's post on slack, maybe we can actually make the above syntax a reality. But I don't know how much effort it's worth? There are less than 200 API methods in all of Meteor core, so it actually might not take that long to go through and document how to use import one, while improving some of the most egregious ones?